app/Support/ConfigStore.php

```
@@ -223,16 +213,62 @@ public static function set(string $path, $value): void
+        $pass = $dbCfg['password'] ?? '';
+        $name = $dbCfg['database'] ?? '';
+        $port = (int) ($dbCfg['port'] ?? 3306);
+        $charset = $dbCfg['charset'] ?? 'utf8mb4';
+        $socket = $dbCfg['socket'] ?? null;
+
+        if ($name === '' || $user === '') {
+            return null;
+        }
+
+        mysqli_report(MYSQLI_REPORT_ERROR | MYSQLI_REPORT_STRICT);
+        $mysqli = new \mysqli($host, $user, $pass, $name, $port, $socket);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '267', column '23'). (undefined)

(MissingImport)